devices.ttl

```diff
@@ -697,3 +697,18 @@ _:polyend-tracker device:typeB true .
 _:polyend-tracker device:uri "https://polyend.com/tracker" .
 _:polyend-tracker device:details "via https://help.polyend.com/639766-Is-the-Polyend-Tracker-MIDI-compatible" .
 
+_:michigan-synth-works-xvi device:make "Michigan Synth Works" .
+_:michigan-synth-works-xvi device:model "XVI" .
+_:michigan-synth-works-xvi device:outputs 1 .
+_:michigan-synth-works-xvi device:typeB true .
```

Comment:
The description on the MiSW site says:

> As of Serial#4161, all devices have TRS MIDI A

Maybe we should record the XVI as `device:typeA` (with a `device:notes` fields saying that models before Serial#4161 were Type B). Or does the XVI accept both types?

Reply:
Ah, good catch. The top-level description still just says B. It doesn't accept both that I'm aware of. I'll update it to `typeA` and add an appropriate note.